tests/test_webhook.py

```diff
@@ -452,6 +453,11 @@ def message_content(event):
         def message_sticker(event):
             self.assertEqual('message', event.type)
             self.assertEqual('sticker', event.message.type)
+            self.assertIn(
+                event.message.sticker_resource_type,
```

Comment:
This is a test checking if an `event` passed to `message_sticker` method is `StickerMessage` and its message is `StickerMessage`.
So, we should remove an `assertIn` sentence.

Reply:
ok